client/client.go

```diff
@@ -318,8 +318,9 @@ type response struct {
 
 // Error is the real value of response.Result when an error occurs.
 type Error struct {
-	Kind    string `json:"kind"`
-	Message string `json:"message"`
+	Kind    string      `json:"kind"`
+	Value   interface{} `json:"value"`
```

Comment:
I left it because it's documented, the server exposes it, this was just the first time it was getting used client-side